OpenSim/Examples/Plugins/BodyDragExample/BodyDragForce.cpp

```diff
@@ -145,7 +145,8 @@ void BodyDragForce::computeForce(const SimTK::State& s,
     // and the force vector itself to be in the body frame
```

Comment:
I'm fairly certain that this is incorrect - the point should be in the body frame and the force in the ground frame. Shall I correct this as well? Or do you not care too much about the dragForce?